tests/integration/test_pip_packages.py

```diff
@@ -31,14 +31,15 @@ def test_all_pip_packages(env_name, test_env, tmpdir):
         payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": ["pip"]}
     )
     completed_response = client.wait_for_complete_request(initial_response)
+    response_data = utils.sort_cachito_response(completed_response.data)
```

Comment:
@ejegrova Do you have any ideas instead of it?

Reply:
@chmeliik What do you think? I think we will have overhead anyway: we still will need to go through all packages to sort dependencies, packages, and dependencies outside of packages.

Reply:
Are we sorting the whole response? AFAICT only `packages`, `dependencies` and `packages[].dependencies` are sorted (which is ok). If you want to avoid overhead, you could sort the data in place instead of recreating it, something like this:
```
sort_in_place(dependencies)
sort_in_place(packages)
for package in packages:
    sort_in_place(package["dependencies"])
```
But that depends if you want to modify the original data or not

Reply:
I think both approaches are equal with the amount of additional code. I just was not sure, maybe there some additional conses.

Reply:
I like Adam's suggestion, that's something similar that I had in mind. You can use the same function for expected data afterwards.